### ISSUE TITLE
Kartographer should use another tile server, or be added to WMF's allowlist

### ISSUE THREAD

**Issue by GreenReaper** [wbstack/mediawiki#69]
[Kartographer-emabled embedded maps and frames](https://www.mediawiki.org/wiki/Help:Extension:Kartographer) do not work on WBStack because WMF now forbids third-party use.

* [Extension:Kartographer](https://www.mediawiki.org/wiki/Extension:Kartographer) was added to WBStack in [1.33-b0.6-c0.8-e0.39](https://github.com/wbstack/mediawiki/blob/main/wbstack/CHANGELOG.md).
* After [an outage in February 2020](https://wikitech.wikimedia.org/wiki/Incident_documentation/2020-02-04_maps) and [an announcement that August](https://lists.wikimedia.org/hyperkitty/list/maps-l@lists.wikimedia.org/message/SNS54LBGFB646LPZ25M2SSDVNPNJ7OLC/), access to WMF's tile server [was limited 14 October 2020](https://phabricator.wikimedia.org/T261424).
* [Affiliates were added to an allowlist](https://phabricator.wikimedia.org/T261694) ([example](https://gerrit.wikimedia.org/r/c/operations/puppet/+/632544/2/modules/varnish/templates/upload-frontend.inc.vcl.erb#192)). `opencura.com` is not in there, so `maps.wikimedia.org` returns 403 Forbidden.

### Solutions
1. The simplest solution is to disable Kartographer; this might mean no globe coordinate or geoshape maps as well as embeds. However, that's basically what WBStack has now, and it would look less broken.
2. [Configure Kartographer](https://phabricator.wikimedia.org/T259868) to use [a free tile server](https://wiki.openstreetmap.org/wiki/Tile_servers). It seems most do not support osm-intl, at least not the way the extension expects; but [it can be made to work](https://phabricator.wikimedia.org/T271617#6953386) ([this item](https://linkedopendata.eu/wiki/Item:Q235272) had URLs like `https://a.tile.openstreetmap.org/12/2138/1468.png?lang=en`).
3. It's theoretically possible to run a tile server, but [there's a lot involved](https://wikitech.wikimedia.org/wiki/Maps).
4. WBStack could be justified under the [the new terms](https://meta.wikimedia.org/wiki/Revised_Maps_Terms_of_Use_(March_2021)). [Movement affiliates](https://phabricator.wikimedia.org/T261424#6419744) are likely to be permitted. However, [WBStack terms](https://www.wbstack.com/terms-of-use) do not limit its usage to affiliates or community purposes. It could be argued it's under the auspices of WMDE and usage will be limited; but others using WBStack code may not be so justified, so it isn't a good default code-wise. Perhaps something that can be enabled as an option if the project affirms compliance, e.g. [Wikibase Registry](https://wikibase-registry.wmflabs.org) if [moved](https://phabricator.wikimedia.org/T213712#6995812) (currently _could_ use it, as wmflabs is enabled), or [other?] Wikibase Community User Group efforts (one reason to remain affiliated).

**Comment by addshore** [comment]
Sounds like this is something that should be unstreamed and fixed in the Wikibase Kartographer code too, as 3rd parties will now start having this issue, and the defaults ideally would work

**Comment by GreenReaper** [comment]
Yes, probably, if it has not been done already - though my thought was for WBStack specifically.

As it happens, I bumped into a member of [Technical Wishes](https://meta.wikimedia.org/wiki/WMDE_Technical_Wishes) at Wikimania 2021 (I think Timur Vorkul?), and I found out [that they have maps as a focus for the coming years](https://meta.wikimedia.org/wiki/WMDE_Technical_Wishes/Geoinformation), having issues with Kartographer already (e.g. conflicts with Flagged Revisions). I pointed them to this issue and asked them to consider whether option 4 is a possibility as well, especially as WBStack is now WMDE-supported.

No idea if anything will result from it (they have plenty on their list already), but I thought I should mention it in case they follow up.
